python/tests/test_aac/plugins/print_defs/test_print_aac_definitions.py

```diff
@@ -28,16 +29,32 @@ def run_print_defs_cli_command_with_args(self, args: list[str]) -> Tuple[int, st
         exit_code = result.exit_code
         std_out = str(result.stdout)
         output_message = std_out.strip().replace("\x1b[0m", "")
-        return exit_code, output_message
+        return exit_code, output_message, std_out
```

Comment:
Not understanding why you need to return `std_out` when you are getting its content within the `output_message` and can perform your assertion checks against `output_message` like we have previously?

Reply:
I was thinking that the output message would just be the message, not the printout.  But you are right, I'm not sure what I was thinking.

Reply:
No worries. The rename does make it seem like something might get lost in the `output_message` vs the original `std_out` instance.